met-api/src/met_api/resources/engagement_metadata.py

```diff
@@ -75,9 +73,16 @@ class EngagementMetadata(Resource):
     @cross_origin(origins=allowedorigins())
     @API.doc(security='apikey')
     @API.marshal_list_with(metadata_return_model)
-    @require_role(VIEW_ENGAGEMENT_ROLES)
+    @_jwt.requires_auth
     def get(engagement_id):
         """Fetch engagement metadata entries by engagement id."""
+        authorization.check_auth(
```

Comment:
Levels of abstraction are tricky here, but the ticket still calls for engagement metadata to allow CRUD operations for users who can edit engagements.

I believe it's the metadata_taxon file that will restrict its operations to the `MANAGE_METADATA` role.

Reply:
Thanks, changed it back to use edit engagement. The only change here being allowing team members assigned to the engagement to edit as well.